ftw/simplelayout/tests/test_contentpage_behavior.py

```diff
@@ -41,3 +42,13 @@ def test_title_shows_if_show_title_is_true(self, browser):
         self.assertEqual(
             ['A page'],
             browser.css('.documentFirstHeading').text)
+
+    @browsing
+    def test_show_hide_title_field_is_only_for_admins(self, browser):
```

Comment:
@jone hmm I guess it would be much simpler to simply ignore the permission with lawgiver... this way the rolemap.xml wins and the test is right. 

Currently I'm unable to say how we will use this feature in the future. The only usecase so far is to only permit Site Admins to edit this field.